frontend/app/services/EventbriteService.scala

```diff
@@ -54,7 +54,7 @@ trait EventbriteService extends WebServiceHelper[EBObject, EBError] {
   def mkRichEvent(event: EBEvent): Future[RichEvent]
   def getFeaturedEvents: Seq[RichEvent]
   def getEvents: Seq[RichEvent]
-  def getTaggedEvents(tag: String): Seq[RichEvent]
+  def getTaggedEvents(tag: String): Seq[RichEvent] = Seq.empty
```

Comment:
I'd *like* to be able to define this just on MasterclassesService, but I couldn't work out how to make that actually work…